test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1042,7 +1041,20 @@ UCS_TEST_P(test_uct_cm_sockaddr, conn_to_non_exist_ip)
     ucs_status_t status;
     size_t size;
 
-    skip_tcp_sockcm();  /* error handling isn't implemented yet in tcp_sockcm  */
+    m_entities.clear();
+
+    /* tcp_sockcm requires setting this parameter to shorten the time of waiting
+     * for the connect() to fail when connecting to a non-existing ip.
+     * A transport for which this value is not configurable, like rdmacm,
+     * uct_config_modify will fail and have no effect. */
+    if (m_cm_config) {
+        /* coverity[check_return] */
+        uct_config_modify(m_cm_config, "SYN_CNT", "1");
+    }
+
+    /* recreate m_server and m_client with the above env parameter changed */
+    init();
```

Comment:
why just dont call `uct_config_modify(m_cm_config, "SYN_CNT", "1");` from `init();` ?

Reply:
so that it would have an effect only for this specific test. didn,t want to change the default behavior of tcp for all the tests here

Reply:
maybe to move it to separate class then? it would be initialized once

Reply:
instead of just calling clear() and init() ?

Reply:
yes, this class would just overload `init()`

Reply:
>yes, this class would just overload init()

yes i understood :) do you think that creating a new class is required here when it's possible to just replace it with two function calls?

Reply:
actually 3 :) and yes, IMO transparent initialization makes test and code clean

Reply:
need separate class which overrides init()

Reply:
only for this test case? (it's the only one that needs to reset the default value of SYN_CNT)

Reply:
yes